packages/ketcher-core/src/utilities/getSvgFromDrawnStructures.tsx

```diff
@@ -5,6 +5,30 @@ const ADDITIONAL_TOP_MARGIN = 54;
 const ADDITIONAL_LEFT_MARGIN = 50;
 const DEFAULT_MARGIN = 10;
 
+const removeHoverEffects = (wrapper: SVGSVGElement): void => {
```

Comment:
This logic is too complicated and too specific, I think you can remove it completely and just add `dynamic-element` class to snake bonds selection, which will be automatically covered by the existing logic